content/docs/studio/get-started.md

```diff
@@ -105,6 +107,13 @@ You should now see that a project has been added in your dashboard.
 
 2. You can also make the project public by [sharing it on the web].
 
+## Submit experiments and track live metrics
+
+1. You can use your project to [submit new experiments] from Iterative Studio.
+
+2. For the experiments that you submit from Iterative Studio, you can [track
+   live metrics][live-metrics] by using [DVCLive].
```

Comment:
```suggestion
2. For the experiments that you submit from Iterative Studio, you can [track
   live metrics] by using [DVCLive].
```

Reply:
The reason I used `[live-metrics]` was for consistency across all the pages. The actual text may be slightly different in different pages, but I wanted to use a consistent name for the link. In fact, I was trying to figure out if/how I can define `[live-metrics]` in one place and use it across multiple pages - is this possible?

Reply:
Yes, naming ref links is most useful when reusing them in links with different words.

Reply:
@jorgeorpinel can I consider this resolved?

Reply:
Feel free to mark things resolved when they are :) Resolving...

Reply:
Feel free to mark things resolved when they are :) Resolving...